engine/Assets/Scripts/Modes/MatchMode.cs

```diff
@@ -1,20 +1,52 @@
 using Synthesis.UI.Dynamic;
 using System.Collections;
 using System.Collections.Generic;
+using SynthesisAPI.Utilities;
 using UnityEngine;
 using TMPro;
+using Logger = SynthesisAPI.Utilities.Logger;
 
 public class MatchMode : IMode {
     //PracticeMode practiceMode = new PracticeMode();
     public static int currentFieldIndex = -1;
     public static int currentRobotIndex = -1;
 
+    private int _redScore = 0;
+    private int _blueScore = 0;
+
     public const string PREVIOUS_SPAWN_LOCATION = "Previous Spawn Location";
     public const string PREVIOUS_SPAWN_ROTATION = "Previous Spawn Rotation";
 
     // Start is called before the first frame update
     public void Start() {
         DynamicUIManager.CreateModal<MatchModeModal>();
+        SynthesisAPI.EventBus.EventBus.NewTypeListener<OnScoreEvent>(
```

Comment:
Looks like this isn't used/tested. Is it better to leave it commented out until we use MatchMode and can test it out?

Reply:
Unless it's causing issues with other things, I think leave it in? I was just trying to add the same things to both practice and match mode to make it easier to integrate later, and I think I tested match mode a bit a while ago (but not with a remotely recent version I don't think).